templates/cli-start/.actor/Dockerfile

```diff
@@ -17,7 +19,11 @@ RUN npm --quiet set progress=false \
     && npm --version \
     && npm cache clean --force
 
-# TODO Install any other dependencies here
+# TODO Install any other dependencies here - you may use `ubi` for that:
```

Comment:
maybe add a link, judging by their star count, the project might not be very well known